routers/routes/routes.go

```diff
@@ -538,16 +517,42 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
-	}, reqSignIn, context.RepoAssignment(), context.UnitTypes())
+	}, reqSignIn, context.RepoAssignment(), context.UnitTypes(), context.LoadRepoUnits(), context.CheckUnit(models.UnitTypeIssues))
+
+	// Releases
+	m.Group("/:username/:reponame", func() {
```

Comment:
Move this for easy add middleware `contex.CheckUnit`. Before this, releases URLs are put on many places.